waflyctl.go

```diff
@@ -1358,7 +1358,7 @@ func WithShieldingCondition(client fastly.Client, serviceID string, version int,
 		Service:   serviceID,
 		Version:   version,
 		Name:      "waf-soc-with-shielding",
-		Statement: "!req.backend.is_shield",
+		Statement: "waf.executed",
```

Comment:
According to federico, waf.executed will be set on both the edge and shield nodes, so this will still send duplicate logs.